Makefile

```diff
@@ -0,0 +1,22 @@
+REPO = github.com/imega/pb-dropbox-downloader
```

Comment:
This project already has [a task runner](https://taskfile.dev/). Using also make is redundant in my opinion.

Reply:
I removed the Makefile. I am not trying to change your mind. I noticed that you have used and support different configurations for CI and Dev environments. In fact it is double work. I tried to help a project that it will be one a config in different environments.

Reply:
No problem. Different configurations for CI and Dev are command practices. In this case it is needed to be able to run cli application locally (without docker on windows machine). And make does not change the situation in any way.